canonical-web-and-design/ubuntu.com
# Update /legal/data-privacy page

## Done

- Update the /legal/data-privacy page with " or by deselecting the "Enable Google Analytics in MAAS UI to shape improvements in user experience” box where applicable."

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/legal/data-privacy
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1AlOE2_-BqZNP4hiiuf8kZney8g1zvqAlDYl2u3X6m6I/edit#)
